generate.sh

```diff
@@ -76,7 +76,7 @@ if [ -z "$ON_HEROKU" ]; then
 fi
 
 # Setting environment variables
-ENVVARS="$(python ${BASE}/modules/scripts/config.py "${OWNER}" "${REPONAME}")"
```

Comment:
OWNER and REPONAME serve as the defaults for author and projectname. It's still needed but now I'm directly reading the value of the environment variable from the script instead of passing as a parameter